vault/resource_aws_secret_backend_test.go

```diff
@@ -159,3 +164,16 @@ resource "vault_aws_secret_backend" "test" {
   region = "us-west-1"
 }`, path)
 }
+
+func testAccAWSSecretBackendConfig_userTemplate(path, accessKey, secretKey, templ string) string {
+	return fmt.Sprintf(`
+resource "vault_aws_secret_backend" "test" {
+  path = %[1]q
```

Comment:
Could we have these as `%s`?

Reply:
Why? in going by golang features. https://zetcode.com/golang/string-format/. using %s is a bit verbose IMO. ill change it if you still want me to.

Reply:
changed to move this along.

Reply:
Sorry, I also meant since we aren't reusing any of the indexed values and `%[1]s` is only used in one spot, we can probably just set them all to be `%s`

Reply:
done